Src/Sarif.PatternMatcher.Function/Sarif.PatternMatcher.Function.csproj

```diff
@@ -13,6 +13,7 @@
     <None Remove="Properties\PublishProfiles\**" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.2.2" />
```

Comment:
>2.2.2 [](http://example.com/codeflow?start=67&length=5)

2.2.2 contains a fix for the vulnerability found in 2.1.0: 
[https://access.redhat.com/security/cve/CVE-2020-1045](https://access.redhat.com/security/cve/CVE-2020-1045)

Reply:
there are no release notes for the PR, which is customer impacting, potentially, because runtime components are changing.

Reply:
Just updated and added some detail in the description